sqldef.go

```diff
@@ -51,12 +52,20 @@ func Run(generatorMode schema.GeneratorMode, db adapter.Database, options *Optio
 		return
 	}
 
+	beforeApply := []string{}
+	for _, query := range strings.Split(options.BeforeApply, ";") {
```

Comment:
This could wrongly split an SQL like `... xxx = ";";`. Can we just NOT split this?

Reply:
addressed in 58561f7